matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/auth/registration/RegistrationWizard.kt

```diff
@@ -76,9 +76,11 @@ interface RegistrationWizard {
     suspend fun dummy(): RegistrationResult
 
     /**
-     * Perform the other stage.
+     * Perform custom registration stage by sending a custom JsonDict
+     * with included current registration "session" param.
```

Comment:
It's not clear reading the doc, that the SDK will actually add the parameter to the dict

Reply:
Updated